api/modelconfig/modelconfig_test.go

```diff
@@ -180,3 +180,28 @@ func (s *modelconfigSuite) TestGetSupport(c *gc.C) {
 	c.Assert(called, jc.IsTrue)
 	c.Assert(level, gc.Equals, "level")
 }
+
+func (s *modelconfigSuite) TestSequences(c *gc.C) {
+	called := false
+	apiCaller := basetesting.APICallerFunc(
+		func(objType string,
+			version int,
+			id, request string,
+			a, result interface{},
+		) error {
+			c.Check(objType, gc.Equals, "ModelConfig")
+			c.Check(id, gc.Equals, "")
+			c.Check(request, gc.Equals, "Sequences")
+			c.Check(a, jc.DeepEquals, nil)
+			results := result.(*params.ModelSequencesResult)
+			results.Sequences = map[string]int{"foo": 5, "bar": 2}
+			called = true
```

Comment:
strictly speaking don't need this because we are returning a specific result to check

Reply:
Not entirely true, because the api implementation could return those results without calling the facade :)